### PR TITLE
[QOL-9055] fix parameter name for 'follow' Pylons routing

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

File: .docker/Dockerfile-py3.ckan [hunk 1]
```diff
@@ -11,7 +12,7 @@ ENV CKAN_INI=/app/ckan/default/production.ini
 WORKDIR "${APP_DIR}"
 
 ENV DOCKERIZE_VERSION v0.6.1
-RUN apk add --no-cache curl build-base git libxml2-dev libxslt-dev postgresql postgresql-client postgresql-dev \
+RUN apk add --no-cache curl build-base git libffi-dev libxml2-dev libxslt-dev postgresql postgresql-client postgresql-dev \
```
Comment (duttonw) [inline .docker/Dockerfile-py3.ckan:15]: interesting https://sourceware.org/libffi/

File: .docker/scripts/init-ext.sh [hunk 2]
```diff
@@ -4,11 +4,33 @@
 #
 set -e
 
+install_requirements () {
+    PROJECT_DIR="$1"
+    for filename in requirements-$PYTHON_VERSION.txt requirements.txt pip-requirements.txt; do
+        if [ -f "$PROJECT_DIR/$filename" ]; then
+            pip install -r "$PROJECT_DIR/$filename"
+            return 0
```
Comment (duttonw) [inline .docker/scripts/init-ext.sh:16]: could you add comment, like
run first file found and break loop.

File: ckanext/datarequests/controllers/ui_controller.py [hunk 3]
```diff
@@ -37,8 +37,8 @@ def comment(self, id):
     def delete_comment(self, datarequest_id, comment_id):
```
Comment (duttonw) [inline ckanext/datarequests/controllers/ui_controller.py:37]: so not changing delete_comment to id also?

**Review by duttonw — APPROVED** [review]

File: ckanext/datarequests/controllers/ui_controller.py (hunk 3 above)
Comment (ThrawnCA) [reply ckanext/datarequests/controllers/ui_controller.py:37]: No. This one uses the names consistently, plus it has two separate IDs that it needs to distinguish.

File: .docker/Dockerfile-py3.ckan (hunk 1 above)
Comment (ThrawnCA) [reply .docker/Dockerfile-py3.ckan:15]: It's not really needed for this particular extension, it's more for ckanext-harvest, but we may as well make our containers as consistent as possible.

**Pull request merged**